pkg/logs/processor/processor.go

```diff
@@ -81,7 +81,7 @@ func applyRedactingRules(msg message.Message) (bool, []byte) {
 				return false, nil
 			}
 		case config.MaskSequences:
-			content = rule.Reg.ReplaceAllLiteral(content, rule.ReplacePlaceholderBytes)
+			content = rule.Reg.ReplaceAllLiteral(content, []byte(rule.ReplacePlaceholder))
```

Comment:
This is not great as it is creating a []byte each time for each message.

Reply:
Agreed, I won't take the refactoring of the config here though.